### ISSUE TITLE
Commas in CPC codes

### ISSUE THREAD

**Issue by marcosmilzo** [SWS-Methodology/faoswsTrade#29]
Need to be checked

**Comment by sebastian-c** [comment]
Examples in 2009 tradeflow output:
21439,9; 24310,01; 21439,04

**Comment by sebastian-c** [comment]
There are also some blank keys. Rows 1873 to 1919 of the 2009 tradeflow output have NA for the partner country. Was this meant to have been removed earlier?

**Comment by sebastian-c** [comment]
The column names of the table in the system is:

```
c("fcl", "fcl_cpc_parlink", "cpc", "cpc_fcl_parlink", 
"description", "suafbs_convfact", "suafbs_notes", "fclcpc_convfact", 
"production_notes")
```

**Comment by marcosmilzo** [comment]
Fix #29 with 8b08fdc7bc256c4807f4c64a326353c9e814f666
